ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/io/Gcs.java

```diff
@@ -103,7 +104,8 @@ private Batch(String bucket, String keyPrefix) {
 
       @Override
       protected CompletableFuture<Void> close() {
-        return batchCloseHook.apply(storage.create(blobInfo, content.toByteArray()));
+        return batchCloseHook.apply(
```

Comment:
It would be worth commenting here that we expect this to throw a particular exception type if the blob exists.